samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb

```diff
@@ -130,7 +171,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can use styling to preserve the preformatting of such content."
+    "You can use styling to preserve the pre-formatting of such content."
```

Comment:
```suggestion
    "You can use styling to preserve the preformatting of such content."
```

Reply:
Thanks! I think my spell checker had flagged that. Haha. Sorry about that. [Fixed here.](https://github.com/dotnet/interactive/pull/2211/commits/080f2de0583fb05eccfb83e3c405801a51e544ab)